src/Castle.Windsor.Tests/LoggingFacility/Log4NetFacilityLognameOverrideTests.cs

```diff
@@ -47,15 +47,15 @@ public void Teardown()
 
 		private IWindsorContainer container;
 
-		[Test]
+		[Test] // Why override.? does not make sense, not what we told the logger to do.
 		public void OverrideTest()
 		{
 			container.Register(Component.For(typeof(SimpleLoggingComponent)).Named("component"));
 			var test = container.Resolve<SimpleLoggingComponent>("component");
 
 			test.DoSomething();
 
-			String expectedLogOutput = String.Format("[INFO ] [Override.{0}] - Hello world" + Environment.NewLine, typeof(SimpleLoggingComponent).FullName);
```

Comment:
"Override" was the log name you removed above, this is the only purpose of this whole test class.

Reply:
Ok I see it now, got confused between what the constructor of the factory was doing and the CreateConfiguredContainer extension.